source/clear-linux/reference/collaboration/writing-guide.rst

```diff
@@ -0,0 +1,419 @@
+
+Use strong verbs
+****************
+
+Passive verbs make writing stuffy and formal. Use strong verbs to get to the
+point and avoid unnecessary words and phrases.
+
+Use imperatives
+===============
+
+Commands, also called imperatives, are the fastest and most direct way of giving
+someone instructions. For example:
+
+**Use this:** ::
+
+  Send it to me.
+
+**Not this:** ::
+
+  I would appreciate it if you would send it to me.
+
+Use present tense
+=================
+
+Use simple present tense instead of future tense for most text. Search for the
+words "will" or "shall" to find future tense instances. Future tense is
+acceptable for conditional statements, such as in a caution or a warning. For
+example:
+
+**Use this:** ::
+
+  The system operates at a nominal temperature of 180 degrees Fahrenheit.
+
+**Not this:** ::
+
+  The system will operate at a nominal temperature of 180 degrees Fahrenheit.
+
+Avoid nominalizations
+=====================
+
+Avoid nominalizations, which are nouns formed from verbs.
+
+For example:
+
+===================== =====================
+ Verb 				         Nominalization
+===================== =====================
+ complete  			       completion
+ provide  			       provision
+ fail  				         failure
+ install  			       installation
+===================== =====================
+
+For example:
+
+**Use this:** ::
+
+  We discussed the matter.
+
+**Not this:** ::
+
+  We had a discussion about the matter.
+
+Or:
+
+**Use this:** ::
+
+  IT has installed the software.
+
+**Not this:** ::
+
+  IT has completed the installation of the software.
+
+Avoid words ending in -ing
+==========================
+
+Avoid using words ending in -ing unless they are part of a technical name. For
+example:
+
+**Use this:** ::
+
+  There is no way to verify this.
+
+**Not this:** ::
+
+  There is no way of verifying this.
+
+Use the active voice
+====================
+
+Use active voice whenever possible to show who or what is performing an
+action.
+
+* Active voice follows standard English word order: SUBJECT–VERB–OBJECT
+  (where the OBJECT is optional).
+* Passive voice reverses the order and weakens the verb: OBJECT–be VERB–by
+  SUBJECT (where the OBJECT is optional).
+
+For example:
+
+**Use this:** ::
+
+  I made a mistake.
+
+**Not this:** ::
+
+  A mistake was made. *(By whom?)*
+
+Or:
+
+**Use this:** ::
+
+  We released version 2.0 in June.
+
+**Not this:** ::
+
+  Version 2.0 was released in June.
+
+Avoid long noun phrases
+***********************
+
+Noun phrases (a noun and other words that describe or modify it) can be
+difficult to understand. Try to limit the number of modifiers in a noun phrase
+to two. For example:
+
+**Use this:** ::
+
+  Integration policies for power management mechanisms.
+
+**Not this:** ::
+
+  Power management mechanism integration policies.
+
+Or:
+
+**Use this:** ::
+
+  Requirements for test desks that measure signal integrity.

```

Comment:
"test desks" here and "test deck" below.

Reply:
Removed second example as it wasn't necessary. :)